tests/integration_tests/core_tests.py

```diff
@@ -537,7 +537,7 @@ def test_mssql_engine_spec_pymssql(self):
         )
 
     def test_comments_in_sqlatable_query(self):
-        clean_query = "SELECT '/* val 1 */' as c1, '-- val 2' as c2 FROM tbl"
+        clean_query = "SELECT\n  '/* val 1 */' AS c1,\n  '-- val 2' AS c2\nFROM tbl"
```

Comment:
[suggesting] it could be good to have some sort of reusable `compare_sql(strict=False, case_sensitive=False, disregard_schema_prefix=True)` function that could be reused in unit tests. Maybe it's a method of `ParseQuery` (`is_identical` or `is_similar`)

Reply:
Yeah, good point, I'll add that!